### ISSUE TITLE
Copy to clipboard button needs tooltip when clicked

### ISSUE THREAD

**Issue by PascalPrecht** [status-im/nim-status-client#1455]
<img width="374" alt="Screenshot 2020-12-02 at 12 01 51" src="https://user-images.githubusercontent.com/445106/100864351-2bd43b80-3496-11eb-97bf-ffe308a2dd5c.png">

Also, "pressed" state isn't needed here. Right now it turns gray when pressed

**Comment by jinhojang6** [comment]
@PascalPrecht An ambassador wants to take this issue. Can I have him start working on this?

**Comment by PascalPrecht** [comment]
Yes sure!

**Comment by staked-smart-ace** [comment]
Hello Pascal
This is Wong. I just introduced by Jinho and want to contribute my codes for this project.
So, I would like to fix this issue first.
May I make a new work branch myself, and new PR after I fixed it?
And then, if you send me other chat channel ID, it would be great for us to keep the session for working.
Regards,
Wong

**Comment by jinhojang6** [comment]
Hey Wong, thank you for contributing to Status Desktop,

I think making a separte branch and submitting a pull request will be great

@PascalPrecht, Is there any Discord server that open source contributors can communicate with the Status desktop team?

**Comment by staked-smart-ace** [comment]
Sure,
I am already joint on Status Discord server.
Which channel should I start to conversation from?

Wong

**Comment by PascalPrecht** [comment]
Hey @staked-smart-ace !
Thanks for taking a stab on this one :)
So on discord you can talk to us in #desktop, otherwise on Status you can find us in #status-desktop

> May I make a new work branch myself, and new PR after I fixed it?

Absolutely! Just fork the repo, clone it, make it run (reach out to us if you need help with that and the instructions aren't enough, and then just send PRs from any branch based off of (latest) `master`

When you create a PR, make sure that the commit(s) in there have a line that says something like

```
Closes #1455
```
So that once your PR is merged and your commit hits the repo, this issue will be closed automatically.
We will also review your PR and ask your to update it if there are any changes we may request.

If you need help with updating your PR, just let us know. 

Looking forward to your contribution!

**Comment by staked-smart-ace** [comment]
@PascalPrecht 
Thanks for reply.
Could you make sure discord server url?
I dont found #desktop channel on there, I am in https://discord.com/invite/3Exux7Y.
If it is not correct, please provide right server url for me to join.
I am looking for hearing back from you soon. :)

**Comment by PascalPrecht** [comment]
Ah you're in the community server!
Okay, then you should use Status and #status-desktop

**Comment by staked-smart-ace** [comment]
yeah, what is your ID on there?

**Comment by PascalPrecht** [comment]
Just join #status-desktop, we're all there! :)

**Comment by staked-smart-ace** [comment]
I just joint and write some messages, are you looking for my messages?

**Comment by staked-smart-ace** [comment]
I fixed this issue. :)
However, I don't have write permission for this repo.
Could you give me the write permission so that I push my updates and make PR?

**Comment by staked-smart-ace** [comment]
And then, I found another issue from this.
Currently, is copied shown all text including "Get Status at ".
Actually, it is right to copy only the URL (https://status.im)
So, I am gonna fix also this issue. :)

**Comment by PascalPrecht** [comment]
> Could you give me the write permission so that I push my updates and make PR?
Hey @staked-smart-ace !
Awesome, great to hear you've got a fix in the making. 
To propose your changes you don't actually need write permission on this repository. What you do is you **fork** this repository (it's another word for cloning it into your GitHub). That fork gives you write permission, so you can then push into your fork as much as you want. Once you think you're done, you can send a Pull Request from your fork (and your branch) to our upstream respository.

**Comment by PascalPrecht** [comment]
> And then, I found another issue from this.

Ah interesting! If you work on more issues, just make sure to create a separate branch for those off of latest `master` and send a separate PR for it (assuming it's indeed a separate thing).

Let me know if you need more help!

**Comment by staked-smart-ace** [comment]
Ahh, I didn't fork this master branch, I only cloned and make own new branch from this. ;)
Let me fork and make PR from my one.
